internal/discover/graphics.go

```diff
@@ -56,6 +56,9 @@ func NewGraphicsMountsDiscoverer(logger logger.Interface, driver *root.Driver, n
 		driver.Root,
 		[]string{
 			"libnvidia-egl-gbm.so.*",
+			"gbm/nvidia-drm_gbm.so",
+			"libnvidia-egl-wayland.so.*",
+			"libnvidia-vulkan-producer.so",
```

Comment:
`libnvidia-vulkan-producer.so` is a symlink which should resolve to `libnvidia-vulkan-producer.so.RM_VERSION`. The latter should be detected by the "standard" driver libraries unless it's at a different location to `libcuda.so.RM_VERSION`. 

Is the issue that the `.RM_VERSION` library is not injected, or that the symlink is not created?

Looking at the driver manifest we have:
```
libnvidia-vulkan-producer.so.535.86.10 0755 OPENGL_LIB NATIVE MODULE:egl
libnvidia-vulkan-producer.so 0000 UTILITY_LIB_SYMLINK NATIVE libnvidia-vulkan-producer.so.535.86.10 MODULE:egl
```

Reply:
Neither are injected. Is it ok if I simply put `"libnvidia-vulkan-producer.so*",`?

Reply:
`Looking at the driver manifest we have:`

Where can I see this?

Reply:
The manifest is available when extracting a driver `.run` file.

> Neither are injected. Is it ok if I simply put "libnvidia-vulkan-producer.so*",?

No, since we need to treat symlinks correctly. I assume that the issue is then that `libnvidia-vulkan-producer.so.<RM_VERSION>` doesn't exist at the same location as `libcuda.so.<RM_VERSION>` on your system. Can you confirm this?

Reply:
I believe so.

Reply:
Symlinks should be created via hook. A bind mount may fail.

I can rewrite this for the vulkan producer and `gbm/nvidia-drm_gbm.so`

Reply:
> Symlinks should be created via hook. A bind mount may fail.

After looking at the code, this is not true here. The driver locator will take care of symlinks and resolve them on the host side.